models/WebsiteSetting.php

```diff
@@ -30,7 +30,7 @@ final class WebsiteSetting extends AbstractModel
 
     protected string $name;
```

Comment:
Should we add a default value here?
So this code will cause an error, right?
```
$setting = new WebsiteSetting();
$setting->getName();
```
Also for type attribute.
Or should we make these also nullable?

Reply:
name is not null in db column https://github.com/pimcore/pimcore/blob/0a779330356418ff0f417f2b83520fb72dbff40d/bundles/InstallBundle/dump/install.sql#L593 so i would not make it nullable.